src/index.ts

```diff
@@ -31,7 +32,7 @@ export function upload(body: any, url = `${PINEAPPLE_URL}/upload`) {
 
 async function sendRequest(url: string, options: any) {
   try {
-    return (await fetch(url, options)).result;
+    return (await fetch(url, { ...options, ...defaultOptions })).result;
```

Comment:
I think we should place `...defaultOptions` first and then `...options`, just in case someone using this package wants to override, not important though

Reply:
`options` is never customisable by user

Reply:
because in this order `options` will be overwritten by `defaultOptions`

Reply:
fixed